build.gradle

```diff
@@ -69,7 +69,7 @@ task integTest(type: Test, dependsOn: [jpi, test]) {
 check.dependsOn integTest
 
 jenkinsPlugin {
-    coreVersion = '2.60.3'
+    coreVersion = '2.143'
```

Comment:
I suggest to even use the next LTS version for an even more improved Java 11 support - https://jenkins.io/changelog-stable/#v2.150.1

Reply:
I changed my mind. Let's keep `2.143`.